framework/src/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -412,6 +408,10 @@ protected String standardizeString(
         }
     }
 
+    public boolean hasDependentAnnotations() {
```

Comment:
Whether or not any qualifier in any hierarchy is a dependent annotation.  In other words, does this type system have any dependent annotations.